lib/components/button.vue

```diff
@@ -15,10 +15,7 @@ const linkPropsNoDefault = {
     href: { type: linkProps.href.type },
     to: { type: linkProps.to.type }
 };
-const noConflictLinkProps = {
-    ...omitLinkProps('disabled', 'href', 'to'),
-    ...linkPropsNoDefault
-};
+const noConflictLinkProps = Object.assign(omitLinkProps('disabled', 'href', 'to'), linkPropsNoDefault);
```

Comment:
Would you want this instead:
```js
const noConflictLinkProps = Object.assign({} omitLinkProps('disabled', 'href', 'to'), linkPropsNoDefault);
```
As object assign tries to modify the first object passed to it.